_posts/2022-10-17-septemeber-rubygems-updates.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Septemeber 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in September.
+
+## RubyGems News
+
+This month in RubyGems, we released RubyGems [v3.3.22](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3322--2022-09-07) and Bundler [v2.3.22](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2322-september-7-2022).
+
+We also shipped the following improvements and fixes (see the changelog for more):
```

Comment:
The list below is part of the above releases, so saying "We also shipped (...)" feels a bit off to me. I'd say "These releases include (...)" or something like that instead.